src/routes/console/organization-[organization]/billing/planSummary.svelte

```diff
@@ -21,7 +21,8 @@
     onMount(async () => {
         const invoices = await sdk.forConsole.billing.listInvoices($organization.$id, [
             Query.limit(1),
-            Query.orderDesc('$createdAt')
+            Query.orderDesc('$createdAt'),
+            Query.equal('status', 'pending')
```

Comment:
It would be good to do a check to make sure there's at least one invoice.

Reply:
Checks are actually in place where it's used 👌